src/JustSaying/JustSaying.csproj

```diff
@@ -11,8 +11,8 @@
     <InternalsVisibleTo Include="JustSaying.UnitTests" Key="$(JustSayingKey)" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.7.0" />
-    <PackageReference Include="AWSSDK.SQS" Version="3.7.0" />
+    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="$(AwsSdkSnsVersion)" />
+    <PackageReference Include="AWSSDK.SQS" Version="$(AwsSdkSqsVersion)" />
```

Comment:
These used a specific version rather than the variable so they could be pinned independently of them being increased for the test projects.